test/assignment.coffee

```diff
@@ -234,7 +234,71 @@ test "destructuring assignment against an expression", ->
   [y, z] = if true then [a, b] else [b, a]
   eq a, y
   eq b, z
-
+  
+test "destructuring assignment with objects and splats: ES2015", ->
+  obj = {a:1, b:2, c:3, d:4, e:5}
+  throws (-> CoffeeScript.compile "{a, r..., s....} = x"), null, "multiple rest elements are disallowed"
```

Comment:
`spread = { a.b... }` [works in Babel](https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=env%2Cstage-0%2Cstage-1%2Cstage-2%2Cstage-3&targets=&browsers=&builtIns=false&debug=false&code=var%20a%2C%20yes%2C%20spread%3B%0Aa%20%3D%20%7B%20b%3A%20%7B%20c%3A%20yes%20%7D%7D%3B%0Aspread%20%3D%20%7B%20…a.b%20%7D%3B).